apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/pac/ClaimResponseTypeV2.java

```diff
@@ -35,7 +35,7 @@
           RdaMcsClaim.class,
           RdaMcsClaim.Fields.mbiRecord,
           RdaMcsClaim.Fields.idrClmHdIcn,
-          RdaMcsClaim.Fields.idrHdrToDateOfSvc,
+          List.of(RdaMcsClaim.Fields.idrHdrToDateOfSvc),
```

Comment:
This isn't in the JIRA issue, and I don't want to change this now.  But I'm thinking it makes sense to also include `idrHdrFromDateOfSvc` here.  Based on some early queries, I don't think it will make much of a difference, but from a readability perspective, I think it will make more sense to future maintainers.

Reply:
Sounds good.  Now that the logic is in place it'd be trivial to add it any time.